src/components/App/App.component.jsx

```diff
@@ -1,4 +1,5 @@
-import React, { useLayoutEffect } from 'react';
+//import React, { useLayoutEffect } from 'react';
```

Comment:
Do not keep commented out code, that's what your Git history is for.

Reply:
Done.